front/src/common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector.ts

```diff
@@ -39,6 +41,7 @@ export const useStoreDataForSpeedLimitByTagSelector = () => {
     }
   }, [error]);
 
+  const speedLimitsByTags = compact(uniq(concat(speedLimitTags, speedLimitsTagsByInfraId)));
```

Comment:
Did you have any acceptance criteria for how to sort the tags ? Like leave them like that where we can guess their endpoints origin or merge all of them and sort them alphabetically ?

Reply:
After discussion with the POs, alphabetical order is better everywhere.
I'll add it to the issue.

Reply:
Done